ee/src/main/java/org/jboss/as/ee/structure/EJBClientDescriptor10Parser.java

```diff
@@ -129,7 +129,7 @@ protected void parseClientContext(final XMLExtendedStreamReader reader, final EJ
     protected void parseEJBReceivers(final XMLExtendedStreamReader reader, final EJBClientDescriptorMetaData ejbClientDescriptorMetaData) throws XMLStreamException {
 
         // initialize the local-receiver-pass-by-value to the default true
-        Boolean localReceiverPassByValue = null;
+        Boolean localReceiverPassByValue = true;
```

Comment:
This will break the reason this was changed to Boolean; to distinguish the explicitly configured value from the default, e.g. 

https://github.com/wildfly/wildfly/blob/master/ejb3/src/main/java/org/jboss/as/ejb3/deployment/processors/EJBClientDescriptorMetaDataProcessor.java#L317

I think the fix is at https://github.com/wildfly/wildfly/blob/master/ejb3/src/main/java/org/jboss/as/ejb3/deployment/processors/EJBClientDescriptorMetaDataProcessor.java#L177, e.g. check passByValue != Boolean.FALSE instead of passByValue == Boolean.TRUE.

The comment at L131 above is wrong though!